app.freelens.Freelens.yaml

```diff
@@ -0,0 +1,111 @@
+id: app.freelens.Freelens
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: freelens
+separate-locales: false
+finish-args:
+  # X11 performance
+  - --share=ipc
+  # We need X11
+  - --socket=fallback-x11
+  # Access to wayland
+  - --socket=wayland
+  # Audio Access
+  - --socket=pulseaudio
```

Comment:
why does it need sound playback?

Reply:
I'll try to disable it. It was copy-paste from other Electron apps